### ISSUE TITLE
JPEG Quality

### ISSUE THREAD

**Issue by SteveEdson** [klep/scanline#22]
Thanks for this, it could be incredibly useful to me, however there's no option to specify a quality. Using the scan functionality built into the preview app, I can save a JPEG at max quality with a file size of ~13mb, this scans the JPEG at ~3mb.

I realise this was intended to be used for archiving documents, but it would be great to have this option so I can scan my photos. (I'm doing post-processing so I need the highest quality when scanning).

Thanks!

**Comment by klep** [comment]
You can specify the scanning resolution with -res. Are you talking about the jpeg compression quality?

**Comment by SteveEdson** [comment]
Yeah, I'm scanning at 600 dpi using -res, but I'm wanting to save as a either a full size JPEG or PNG at 100% quality
